selfdrive/ui/qt/offroad/wifiManager.cc

```diff
@@ -279,6 +283,31 @@ QVector<QDBusObjectPath> WifiManager::get_active_connections() {
   return conns;
 }
 
+bool WifiManager::isKnownNetwork(const QString &ssid) {
+  for(QDBusObjectPath path : list_connections()) {
```

Comment:
this loop already exists twice, abstract into a function

Reply:
I did the refractor in the other PR but I can do it here

Reply:
yeah, just abstract into a function